plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineWizardPage.java

```diff
@@ -48,6 +49,9 @@
  */
 public abstract class AppEngineWizardPage extends WizardNewProjectCreationPage {
 
+  private static final String PLUGIN_ID = 
+      "com.google.cloud.tools.eclipse.appengine.newproject.AppEngineStandard"; //$NON-NLS-1$
```

Comment:
I know this is from the old code, but this is weird. `PLUGIN_ID` is usually same as a bundle name.

```
$ git grep 'PLUGIN_ID = '
plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/Activator.java:  public static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.appengine.localserver";
plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/JavaPackageValidator.java:  private static final String PLUGIN_ID = 
plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/DataflowCorePlugin.java:  public static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.dataflow.core"; //$NON-NLS-1$
plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/DataflowUiPlugin.java:  public static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.dataflow.ui"; //$NON-NLS-1$
plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/areas/FieldEditorWrapper.java:  private static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.preferences";
third_party/com.google.cloud.tools.eclipse.jst.server.core/src/com/google/cloud/tools/eclipse/jst/server/core/BasePublishOperation.java:  private static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.jst.server.core";
```